WcaOnRails/app/assets/stylesheets/incidents.scss

```diff
@@ -0,0 +1,62 @@
+// Place all the styles related to the Incidents controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+.incident-form {
+  .nested-fields {
+    .comments-input {
+      @extend .form-control;
+      display: inline-block;
+      vertical-align: middle;
+      width: 50%;
+      margin-right: 15px;
+      margin-left: 15px;
+    }
+    .competition-selector {
+      width: 30%;
+      vertical-align: middle;
+      display: inline-block;
+    }
+  }
+}
+
+.incident-tag,
+.competition-tag {
+  display: inline-block;
+  a.no-after {
+    all: unset;
+    &[target=_blank]::after {
+      content: none;
+    }
+  }
+}
+
+.incidents-log-table-container {
+  .table-responsive {
+    overflow-x: visible;
```

Comment:
Because of the tag filter input:
without this:
![incident-overflow](https://user-images.githubusercontent.com/1007485/30455707-3a44e8b0-99a1-11e7-8fdc-b8610ae7045e.png)
with this:
![incident-overflow-visible](https://user-images.githubusercontent.com/1007485/30455712-3f599e7c-99a1-11e7-8d62-b14600551d6b.png)